draft-ietf-rtcweb-jsep.xml

```diff
@@ -2563,25 +2563,54 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
           <xref target="I-D.ietf-ice-trickle"></xref>, Section 4, is
           only included if such an option was present in the offer.</t>
 
-          <t>The next step is to generate session-level lip sync groups
-          as defined in
-          <xref target="RFC5888" />, Section 7. For each group of type
-          "LS" present in the offer, determine which of the local
-          RtpTransceivers identified by that group's mid values
-          reference a common local MediaStream (as specified in the
-          addTrack and addTransceiver methods). If at least two such
-          RtpTransceivers exist, a group of type "LS" with the mid
-          values of these RtpTransceivers MUST be added. Otherwise,
-          this indicates a difference of opinion between the offerer
-          and answerer regarding lip sync status, and as such, the
-          offered group MUST be ignored and no corresponding "LS" group
+          <t>The next step is to generate session-level lip sync
+          groups as defined in <xref target="RFC5888" />, Section
+          7. For each group of type "LS" present in the offer,
+          determine which of the local RtpTransceivers identified by
+          that group's mid values reference a common local MediaStream
+          (as specified in the addTrack and addTransceiver
+          methods). If at least two such RtpTransceivers exist, a
```

Comment:
I think this is not terribly clear - it talks about existing associations between transceivers and LS groups, while I think we should be talking about what we do when deciding to associate a track with a particular transceiver.

This means that the language needs to go into section 5.9 "applying a remote offer", not here - this section should just say "If the ID value of the msid attributes of the m-lines covered by an LS group differ, the answer MUST NOT contain that LS group".